zstacklib/zstacklib/network/bridge/operations.py

```
@@ -0,0 +1,233 @@
+    if interface:
+        master = get_interface_master(interface)
+        if master and master != name:
+            raise InterfaceOccupiedError(interface, master)
+    
+    if not is_bridge(name):
+        result = subprocess.run(["brctl", "addbr", name], capture_output=True, text=True)
+        if result.returncode != 0:
+            raise BridgeError(f"Failed to create bridge '{name}': {result.stderr}")
+    
+    stp_setting = "on" if stp else "off"
+    subprocess.run(["brctl", "stp", name, stp_setting], check=False)
```

Ruff (0.15.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)